test/ui/ui-accuracy.pool.burning.test.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ *
+ * @group ui
+ */
+import { Keyring } from "@polkadot/api";
+import BN from "bn.js";
+import { WebDriver } from "selenium-webdriver";
+import { getApi, initApi } from "../../utils/api";
+import { Assets } from "../../utils/Assets";
+import { FIVE_MIN } from "../../utils/Constants";
+import { ExtrinsicResult } from "../../utils/eventListeners";
+import { Mangata } from "../../utils/frontend/pages/Mangata";
+import { BrunLiquidityModal } from "../../utils/frontend/pages/BrunLiquidityModal";
+import { Sidebar } from "../../utils/frontend/pages/Sidebar";
+import { DriverBuilder } from "../../utils/frontend/utils/Driver";
+import {
+  setupAllExtensions,
+  takeScreenshot,
+} from "../../utils/frontend/utils/Helper";
+import { createPool } from "../../utils/tx";
+import { getEventResultFromTxWait } from "../../utils/txHandler";
+import { User } from "../../utils/User";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+
+jest.setTimeout(FIVE_MIN);
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+let driver: WebDriver;
+
+//Required env vars:
+//export API_URL=wss://staging.testnode.mangata.finance:9945
+//export UI_URL='https://staging.mangata.finance/' ;
+//export MNEMONIC_META='dismiss .. trumpet' ( Ask Gonzalo :) )
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+let firstCurrency: BN;
+let secondCurrency: BN;
+
+describe("Accuracy tests:", () => {
+  let testUser1: User;
+  let sudo: User;
+  let keyring: Keyring;
+  const visibleValueNumber = Math.pow(10, 19).toString();
+
+  beforeAll(async () => {
+    try {
+      getApi();
+    } catch (e) {
+      await initApi();
+    }
+
+    keyring = new Keyring({ type: "sr25519" });
+    driver = await DriverBuilder.getInstance();
+
+    const { mnemonic } = await setupAllExtensions(driver);
+
+    testUser1 = new User(keyring, undefined);
+    testUser1.addFromMnemonic(keyring, mnemonic);
+
+    sudo = new User(keyring, sudoUserName);
+    await sudo.mint(new BN(0), testUser1, new BN(visibleValueNumber));
+  });
+  test.each([
+    [
+      new BN(Math.pow(10, 19).toString()).sub(new BN(1)).div(new BN(4)), //2.499999X
+      new BN(Math.pow(10, 19).toString()).div(new BN(2)), //5Y
+      "2",
+      "0.050",
+      "0.100",
+    ],
```

Comment:
@Andkoo  feel free to suggest any test scenario! I am worried about rounding stuff :)

Reply:
Do I get it correctly that these tests use values `2` and `1` as the value of the input for burn amount percentage? Because if so, then we ran into problems with value `100` when there were like `0.000000000000000041` liquidity left in the pool - it displayed `0.000% share of the pool` to the user, because of rounding and it was all just confusing.

Reply:
yup! I think that decimals are nowhere well represented in the UI.
I have been talking with Lukas about this and created a task about pools/swapp, but maybe we can add there this issue also,
https://trello.com/c/OrJLadmU
do you mind adding it?